assets/css/base/gutenberg-editor.scss

```
@@ -303,12 +303,11 @@ table {
 
 // Main content width
```

What do you think if we add a comment about this fix here? For example in this way:
```suggestion
// Main content width
// These two rules are copied from https://github.com/woocommerce/storefront/issues/1320#issuecomment-868549528
```

See these numbers so specific, it is a rare thing, so I guess that it is better to give a little bit of context